src/main/java/hudson/remoting/ChannelBuilder.java

```diff
@@ -271,10 +272,10 @@ public ChannelBuilder withProperty(Object key, Object value) {
     }
 
     /**
-     * @since TODO
+     * @since 2.47
```

Comment:
I would document that the backing map may be subject to concurrent modifications or (better) provide a hint/reminder that the builder is not threadsafe